src/lib/tabs/tab-body.html

```diff
@@ -1,6 +1,7 @@
 <div class="mat-tab-body-content" #content
      [@translateTab]="_position"
      (@translateTab.start)="_onTranslateTabStarted($event)"
-     (@translateTab.done)="_onTranslateTabComplete($event)">
+     (@translateTab.done)="_onTranslateTabComplete($event)"
+     cdkScrollable>
```

Comment:
FYI I dont think this is the correct spot for this, I tested this out and it did not seem to fix the issue.
However, I did move cdkScrollable to the tab-group.html where the <mat-tab-body> is and that does seem to solve it.

Reply:
At least in our dev app, this is the element that ends up being scrollable. Did you override any of the default styles in your app?

Reply:
This may be affected by this PR in the future https://github.com/angular/material2/pull/6861

Seems the scrolling container should be a parent of the tab content

Reply:
It looks like it probably would, let's defer this one until #6861 gets in.